src/App.js

```diff
@@ -302,6 +307,20 @@ class App extends Component {
                 <Route exact path="/" component={Home} />
                 <Route path="/alarms" component={AlarmsApp} />
                 <Route path="/data_management" component={DataManagementApp} />
+                
+                <Switch>
+                  <Route exact path="/personal_api_keys" component={PersonalApiKeysTable} />
```

Comment:
As Casper mentioned, this personal_api_keys shows up on the navigation tab as `Personal API_KEYS` with underscore.

Reply:
done here:
https://github.com/nens/lizard-management-client/pull/596